WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopover.swift

```diff
@@ -1,10 +1,24 @@
 import SwiftUI
 
 struct CompliancePopover: View {
-    @State
-    private var isAnalyticsOn = true
+    var goToSettingsAction: (() -> ())?
+    var saveAction: (() -> ())?
+    var shouldScroll: Bool = false
+
+    @StateObject
+    var viewModel: CompliancePopoverViewModel
 
     var body: some View {
+        if shouldScroll {
+            ScrollView(showsIndicators: false) {
+                contentVStack
+            }
+        } else {
+            contentVStack
+        }
+    }
```

Comment:
🟠 I see that this view is supposed to scroll when `shouldScroll` is `true` but I can't scroll the popup when fonts are large. 

https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/175d349e-099b-4551-a7c5-87874f8a301b